src/promptflow/dev-connections.json.example

```diff
@@ -63,6 +63,19 @@
       "key1"
     ]
   },
+  "custom_strong_type_connection": {
+    "type": "CustomConnection",
+    "value": {
+      "api_key": "<your-key>",
+      "api_base": "This is my first custom connection.",
+      "azureml.connection.custom_type": "MyFirstConnection",
```

Comment:
why is this starts with azureml? Is this for azure version only?

Reply:
This is not for azure only. I just want to add a system level prefix, so that it will not mix up with the user defined keys

> why is this starts with azureml? Is this for azure version only?

Reply:
updated to "promptflow.custom.connection.custom_type" and "promptflow.custom.connection.module"